src/store/users/index.ts

```diff
@@ -0,0 +1,37 @@
+import { createAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { FetchUsersPayload } from './saga';
+
+export interface UserMentions {
```

Comment:
this interface shouldn't be at the level of the user, as this is a very local way of thinking about a user. it is tied directly to how users are considered from a very local component. if this interface is important, it shouldn't really bleed outside of the components that are